serum-390/soen390-team16-godzilla
# Inventory Frontend and Update operation

# Description

1) implemented update operation for inventory
2) implemented the front end for the inventory [ features : view ,delete, insert, update items ]

## Link to work item

Include a link to the work item E.g.: #1

## How to Test
Run the application.
In the inventory page : all the items in the inventory db must be in the list
Insert : Insert a new item [ provide all the fields and respect the types]  -> check if the new item is in the table and in db
update : click on Edit next to each row, -> update the fields you want [ you can leave the rest blank] -> after clicking update, the fields should be update in table and db
delete: click on db next to each row -> the item will be deleted from table and db
